doc: Update secure bootloader note for Asset Tracker v2 (#7809)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

.. note::

>    This application supports the :ref:`ug_bootloader` (also called immutable bootloader), which has been enabled by default since the v2.0.0 release.

Reviewer: ```suggestion
   This application supports the :ref:`ug_bootloader` (also called immutable bootloader), which has been enabled by default since the |NCS| v2.0.0 release.
```

>    Devices that do not already have the immutable bootloader can't be firmware upgraded over the air to use the immutable bootloader.

Reviewer: ```suggestion
   Devices that do not already have the immutable bootloader can not be firmware upgraded to use the immutable bootloader.
```
```suggestion
   Devices that do not already have the immutable bootloader can not be firmware upgraded over the air to use the immutable bootloader.
```

>    Devices that do not already have the immutable bootloader can't be firmware upgraded over the air to use the immutable bootloader.

Reviewer: ```suggestion
   Devices that do not already have the immutable bootloader can not be firmware upgraded over the air to use the immutable bootloader.
```

>    Devices that does not already have the immutable bootloader can't be firmware upgraded over the air to use the immutable bootloader.

Reviewer 2: ```suggestion
   Devices that do not already have the immutable bootloader can't be firmware upgraded over the air to use the immutable bootloader.
```
